packages/driver/src/cy/commands/actions/type.ts

```diff
@@ -404,8 +421,13 @@ export default function (Commands, Cypress, cy, state, config) {
           // https://github.com/cypress-io/cypress/issues/19541
           // Send click event on type('{enter}')
           if (sendClickEvent) {
-            // Firefox sends a click event automatically.
-            if (!Cypress.isBrowser('firefox')) {
+            if (
```

Comment:
I find this conditional fairly hard to read, split as it is by comments.
```
// Earlier in file
isFirefoxBefore98 = () => (Cypress.isBrowser('firefox') && Cypress.browserMajorVersion() < 98)

// Firefox versions before 98 always sends a click event automatically.
// Firefox 98+ only send click events automatically if the element is a <button>, but not for <input>s.
if (sendClickEvent && (!isFirefoxBefore98() || $elements.isInput(el))) {
  fireClickEvent(el)
}
```

Reply:
Done. 

But I defined `isFirefoxBefore98` to be a `const` because its browser version doesn't change on runtime.